### PR TITLE
Update app's level

### PR DESCRIPTION
**Pull request by ericgaspar**

### CODE REVIEW

File: apps.json [hunk 1]
```diff
@@ -3643,10 +3651,10 @@
     "proxitok": {
         "category": "social_media",
         "level": 7,
-        "state": "notworking",
```
Comment (ericgaspar) [inline apps.json:3646]: This was added on purpose. Proxytok is not working

**Review by tituspijean — COMMENTED** [review]
- [ ] Flarum should be level 8 (connection error in last test): https://ci-apps.yunohost.org/ci/job/11776

File: apps.json [hunk 2]
```diff
@@ -1506,7 +1510,7 @@
             "arbitrary-limitations"
         ],
         "category": "publishing",
-        "level": 8,
+        "level": 1,
```
Comment (tituspijean) [inline apps.json:1513]:
```suggestion
        "level": 8,
```

as per https://ci-apps.yunohost.org/ci/job/12598

**Review by tituspijean — CHANGES_REQUESTED** [review]

File: apps.json [hunk 3]
```diff
@@ -1051,11 +1055,11 @@
         "url": "https://github.com/YunoHost-Apps/elabftw_ynh"
     },
     "elasticsearch7": {
-        "category": "dev",
-        "state": "working",
         "antifeatures": [
             "not-totally-free"
         ],
+        "category": "dev",
+        "state": "working",
```
Comment (tituspijean) [inline apps.json:1062]:
```suggestion
        "category": "dev",
        "level": 6,
        "state": "working",
```

as per https://ci-apps.yunohost.org/ci/job/12594

File: apps.json [hunk 4]
```diff
@@ -2094,7 +2099,7 @@
     },
     "kimai2": {
         "category": "productivity_and_management",
-        "level": 8,
+        "level": 0,
```
Comment (tituspijean) [inline apps.json:2102]:
```suggestion
        "level": 6,
```

as per https://ci-apps.yunohost.org/ci/job/12512
Upgrades tests will most likely fail, to be fixed in the repo.

**Review by tituspijean — CHANGES_REQUESTED** [review]

**Review by tituspijean — APPROVED** [review]

**Pull request merged**